src/lib/frontend/models.mli

```diff
@@ -30,16 +30,17 @@
 
 (** {1 Models module} *)
 
+type t = {
+  propositional : Expr.Set.t;
+  constants : ModelMap.t;
+  functions : ModelMap.t;
+  arrays : ModelMap.t;
+}
+
 (** Print the given counterexample on the given formatter with the
     corresponding format set with Options.get_output_format.
     - functions: the functions of the model;
     - constants: the variables of the model;
     - arrays: (experimental) the arrays of the model.
 *)
-val output_concrete_model :
-  Format.formatter ->
-  Expr.Set.t ->
-  functions:ModelMap.t ->
-  constants:ModelMap.t ->
-  arrays:ModelMap.t ->
-  unit
+val output_concrete_model : Format.formatter -> t -> unit
```

Comment:
As we use `Fmt`, we can replace this signature by `t Fmt.t`.

Reply:
Sure. Want to do it?